### PR TITLE
TOFCMOS-526

### PR DESCRIPTION
**Pull request by rbudai98**
Bringing Gesture Recognition example from aditof_sdk

### CODE REVIEW

File: bindings/python/examples/gesture_rec/notebook.py [hunk 1]
```diff
@@ -0,0 +1,254 @@
...
+mode_options = [
+        "sr-qnative",
+        "lr-qnative",
+        "sr-native",
+        "lr-native"
+]
+
+# Global variables for the program
+config_options_chosen = config_options[0]
+mode_options_chosen = mode_options[0]
```
Comment (SeptimiuVana) [inline bindings/python/examples/gesture_rec/notebook.py:25]: can we parse the mode, config file and ip via CLI?

Comment (rbudai98) [reply bindings/python/examples/gesture_rec/notebook.py:25]: You can find OptionList-s with Dropdown capability to select the desired one

**Comment by rbudai98** [comment]
v2:
* removed extra cp lines

**Comment by rbudai98** [comment]
v3:
* code formatted

UI:
image

**Review by dNechita — APPROVED** [review]

**Pull request merged**